akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -96,208 +105,222 @@ private[http] object HttpServerBluePrint {
         // `buffer` will ensure demand and therefore make sure that completion is reported eagerly.
         .buffer(1, OverflowStrategy.backpressure)
 
-    // we need to make sure that only one element per incoming request is queueing up in front of
-    // the bypassMerge.bypassInput. Otherwise the rising backpressure against the bypassFanout
-    // would eventually prevent us from reading the remaining request chunks from the transportIn
-    val bypass = Flow[RequestOutput].collect {
-      case r: RequestStart      ⇒ r
-      case m: MessageStartError ⇒ m
-    }
-
     val rendererPipeline =
       Flow[ResponseRenderingContext]
-        .recover(errorResponseRecovery(log, settings))
         .via(Flow[ResponseRenderingContext].transform(() ⇒ responseRendererFactory.newRenderer).named("renderer"))
         .flatMapConcat(ConstantFun.scalaIdentityFunction)
         .via(Flow[ResponseRenderingOutput].transform(() ⇒ errorLogger(log, "Outgoing response stream error")).named("errorLogger"))
 
-    BidiFlow.fromGraph(FlowGraph.create(requestParsingFlow, rendererPipeline, oneHundredContinueSource)((_, _, _) ⇒ ()) { implicit b ⇒
-      (requestParsing, renderer, oneHundreds) ⇒
+    BidiFlow.fromGraph(FlowGraph.create(requestParsingFlow, rendererPipeline)(Keep.none) { implicit b ⇒
```

Comment:
passing parameters here is only for the purpose of using their materialized values—it would seem that requestParsing and rendererPipeline might as well be imported like all the others below